phplintinstall

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
```

Comment:
One thing that is considered a best practice for bash scripts is to add a `set -e` option right after the `#!/bin/bash` line. This will make your script stop immediately after one of the commands fail (i.e. a command returns a non-zero return code)